pkg/config/config.go

```diff
@@ -347,6 +355,160 @@ func (m *Metric) validateMetric(metricIdx int, parent string, discovery *JobLeve
 	return nil
 }
 
+func (c *ScrapeConf) toModelConfig() model.JobsConfig {
+	jobsCfg := model.JobsConfig{}
+	jobsCfg.StsRegion = c.StsRegion
+
+	for _, discoveryJob := range c.Discovery.Jobs {
+		job := model.DiscoveryJob{}
+		job.Regions = discoveryJob.Regions
+		job.Type = discoveryJob.Type
+		job.DimensionNameRequirements = discoveryJob.DimensionNameRequirements
+		job.RoundingPeriod = discoveryJob.RoundingPeriod
+		job.RecentlyActiveOnly = discoveryJob.RecentlyActiveOnly
+		job.Statistics = discoveryJob.Statistics
+		job.Period = discoveryJob.Period
+		job.Length = discoveryJob.Length
+		job.Delay = discoveryJob.Delay
+		job.NilToZero = discoveryJob.NilToZero
+		job.AddCloudwatchTimestamp = discoveryJob.AddCloudwatchTimestamp
+
+		job.ExportedTagsOnMetrics = []string{}
+		if len(c.Discovery.ExportedTagsOnMetrics) > 0 {
+			svc := SupportedServices.GetService(job.Type)
+			if exportedTags, ok := c.Discovery.ExportedTagsOnMetrics[svc.Namespace]; ok {
+				job.ExportedTagsOnMetrics = exportedTags
+			} else if exportedTags, ok := c.Discovery.ExportedTagsOnMetrics[svc.Alias]; ok {
+				job.ExportedTagsOnMetrics = exportedTags
+			}
+		}
```

Comment:
This is the code which fixes https://github.com/nerdswords/yet-another-cloudwatch-exporter/issues/1160 right? 

I'm a bit surprised that this field exists at the root of discovery (I think you mentioned this elsewhere as well). I guess you could have multiple discovery jobs for the same service but that seems more like an exception vs the normal.

Reply:
> I guess you could have multiple discovery jobs for the same service but that seems more like an exception vs the normal.

Yes I think that is more of an exception -- that case however should be covered by this transformation loop.